server/lib/matchmaking/matchmaking-service.ts

```diff
@@ -259,13 +278,13 @@ export class MatchmakingApi {
             // TODO(tec27): We probably shouldn't be blindly sending error messages to clients
             reason: err && err.message,
           })
-          this.handleLeave(client)
+          this.removeClientFromMatchmaking(client)
         }
       }
     },
   }
 
-  matchmakerDelegate = {
+  private matchmakerDelegate = {
```

Comment:
Not a problem you caused, but would probably be good to add explicit types for this + `gameLoaderDelegate`, think error messages would be better if the type definition changes